src/formatter/templates/html_div_formatter.ts

```diff
@@ -44,7 +44,7 @@ export default (
                 ${ when(isChordLyricsPair(item), () => `
                   <div class="column">
                     ${ keep([renderChord(item.chords, line, song, key)], ([renderedChord]) => `
-                      <div class="chord"${ renderedChord ? fontStyleTag(line.chordFont) : '' }>${ renderedChord }</div>
+                      <div class="chord"${ renderedChord ? fontStyleTag(line.chordFont) : '' }>${ renderedChord } </div>
```

Comment:
Does this change also belong in this PR, or is it accidental?

Reply:
Semi-accidental in this PR. It was a fix to create better spacing between trailing chords as well as add some space between chords in a rhythm only section so it doesn't render like this:

```
[Eb5][/][/][/][|][/][/][/][/]
```
![CleanShot 2022-12-12 at 09 20 22](https://user-images.githubusercontent.com/23046374/207111341-86a066c9-07b7-42bc-bb78-12aeb84640d3.png)

I had started this PR a while ago: https://github.com/bettermusic/ChordSheetJS/pull/259 which was for the same thing, this was just a quick approach to make the studio renderer feel more realistic.